src/core/ngx_cycle.h

```diff
@@ -60,6 +60,9 @@ struct ngx_cycle_s {
     ngx_connection_t         *connections;
     ngx_event_t              *read_events;
     ngx_event_t              *write_events;
+#if (NGX_SSL)
+    ngx_event_t              *async_events;
```

Comment:
why no OPENSSL_VERSION_NUMBER

Reply:
In ngx_core.h
```
#include <ngx_cycle.h>
...
#include <ngx_event_openssl.h>

```
When precompile the 'ngx_cycle.h', the macro 'OPENSSL_VERSION_NUMBER' has not been prepared, so it's meanless to add the macro to here.    

Also, the number of objects defined by ngx_cycle_t is limited, i think it's acceptable to add a element into ngx_cycle_t directly.

Reply:
fixed